packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -377,21 +375,30 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
   // Initialization on mount - update array index to 0 (ready state).
   // Only fire on mount (no deps).
   useEffect(() => {
-    if (virtualizerStartIndex < 0) {
+    if (actualIndex < 0) {
       batchUpdateNewIndex(0);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   // If the user passes in an updated renderChild function - update current children
   useEffect(() => {
-    if (virtualizerStartIndex >= 0) {
-      updateChildRows(virtualizerStartIndex);
+    const currentIndex = actualIndex;
+    if (currentIndex >= 0) {
+      updateChildRows(currentIndex);
```

Comment:
```suggestion
    if (actualIndex >= 0) {
      updateChildRows(currentIndex);
```

why is the assignment to `currentIndex` necessary? I can't see `actualIndex` being mutated anywhre in the effect